pkg/stacks/steps_test.go

```diff
@@ -0,0 +1,54 @@
+/*
+Copyright 2019 The Crossplane Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package stacks
+
+import (
+	"testing"
+)
+
+func Test_iconStep_naive(t *testing.T) {
```

Comment:
https://golang.org/doc/effective_go.html#mixed-caps

I think there's some precedent for underscores in test names in the Crossplane codebase, but you should be aware that I shed a single tear each time we introduce a new one. 😢

Reply:
Agreed, that style bothers me a bit too.  I seem to recall a commit in the history of this repo where we fixed all the test names, but I don't think I'd be able to find that easily.

Reply:
I'll ditch the `_naive` test type.  I originally intended to have a table based test as an alternative (or the only test), so I left the good name reserved for future use.